docs/public/styles/index.css

```diff
@@ -74,6 +74,9 @@ html[theme='dark'] {
 
 	--header-accent-height: 0.1875rem;
 }
+html[theme='dark'] .icon {
+	filter: invert(1);
```

Comment:
We need to add this to the "prefers-color-scheme: dark" block too

Reply:
It's there:

https://github.com/preactjs/wmr/pull/503/files#diff-c55bbfb39ca79e1c551b60cf79f7dcb1c40fba2788af584db3e739913cbe9f38R47

Reply:
whoops, my bad 🙈